src/CSFML/Graphics/CircleShape.cpp

```diff
@@ -243,13 +228,7 @@ sfColor sfCircleShape_getFillColor(const sfCircleShape* shape)
     sfColor color = {0, 0, 0, 0};
     CSFML_CHECK_RETURN(shape, color);
 
-    sf::Color sfmlColor = shape->This.getFillColor();
-    color.r = sfmlColor.r;
-    color.g = sfmlColor.g;
-    color.b = sfmlColor.b;
-    color.a = sfmlColor.a;
-
-    return color;
+    return convert(shape->This.getFillColor());
```

Comment:
I find `convert` on its own problematic, as seen here, you'll need to know that we're not turning the color into something different. I'd personally never use such a generic name for a free function.